#Path

#Grids (modulo must be 0 for square size | dimensions)
#Lower square sizes will greatly increase DB size for culling / possible speed
#The frame buffer can be thought of as the viewport
FrameBufferWidth: 1920
FrameBufferHeight: 1080
SquareSize: 12

#Meta

Name: gate
RenderDistance: 10.0

#Optimization
#FPS (0 means as fast as possible)
#Linearization, higher is more simple

Fps: 0
gSLICrOn: false
ContourLinearizationEpsilon: 3.0
MinimumContourArea: 1500

#Background
BackgroundR: 255
BackgroundG: 255
BackgroundB: 255

#Transforms (angle in degrees. scale as decimal percent.)
RelativeTranslateX: 0
RelativeTranslateY: 0
RelativeTranslateZ: 0
UniformScale: 0.4
InitialRotX: 0
InitialRotY: 0
InitialRotZ: 0

#Angle ranges (negatives are accepted) relative to initial transforms
#Start and end values cannot be the same
StartRotateX: -80
EndRotateX: -40
StartRotateY: -20
EndRotateY: 20
StartRotateZ: -10
EndRotateZ: 10
RotateStep: 6

#Filters
#Filters cannot fully contain each other
# http://colorizer.org/ [You're welcome]
FilterCount: 2

#Red
Filter_0_Mode: range
Filter_0_LowerL: 50
Filter_0_LowerA: 138
Filter_0_LowerB: 138
Filter_0_UpperL: 255
Filter_0_UpperA: 255
Filter_0_UpperB: 255

#Black
Filter_1_Mode: range
Filter_1_LowerL: 0
Filter_1_LowerA: 100
Filter_1_LowerB: 100
Filter_1_UpperL: 30
Filter_1_UpperA: 150
Filter_1_UpperB: 150
